### PR TITLE
WSTEAMA-557: Consume ATI data from BFF for CPS Pages

### PR DESCRIPTION
**Pull request by alex-magana**
Resolves **WSTEAMA-557**

Overall changes
======

Migrate STY, MAP, PGL and CSP assets to utilise the ATI analytics values from the BFF.

Code changes
======

- Consume the BFF ATI reporting configuration for CPS (STY, MAP, PGL and CSP) assets.
- Use the BFF contentType ATI configuration for Optimo articles.
    - Article [contentType](https://github.com/bbc/simorgh/pull/10990/files#diff-296dbb4e4081ad707dd31a8041aa141df69f063e16104b53d535d9c5e4a9d7f6R35)
- Update data fixtures.
- Update tests and use the customRender utility for:
    - src/app/components/ATIAnalytics/index.test.tsx
    - src/app/hooks/useClickTrackerHandler/index.test.jsx
- Remove the redundant CPS buildParams functions.
- Update snapshots based on the `shouldRenderLastUpdated` check.
    - [shouldRenderLastUpdated](https://github.com/bbc/simorgh/blob/6af27a0b7ca96db57d057c1e743549f21418fc17/src/app/lib/utilities/filterPopularStaleData/isDataStale.js#L18) check.
- Remove a duplicate secondaryColumn property in the Persian STY fixture.
    - src/app/pages/StoryPage/fixtureData/persian.json

Testing
======

http://localhost:7080/gahuza/23313911
http://localhost:7080/kyrgyz/23292889
http://localhost:7080/mundo/noticias-internacional-51266689

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]
This is great stuff, and so much easier to review the tests now that the inline snapshots have been replaced!

File: data/news/articles/c0g992jmmkko.json [hunk 1]
```diff
@@ -895,11 +895,12 @@
       "metadata": {
         "id": "urn:bbc:ares::article:c0g992jmmkko",
         "atiAnalytics": {
-          "categoryName": "Wedding%20of%20Prince%20Harry%20and%20Meghan%20Markle~Duchess%20of%20Sussex~Hackney~Windsor",
+          "categoryName": "Wedding+of+Prince+Harry+and+Meghan+Markle~Duchess+of+Sussex~Hackney~Windsor",
```
Comment (HarveyPeachey) [inline data/news/articles/c0g992jmmkko.json:898]: Does getting rid of the space encoding here affect anything?

**Review by shayneahchoon — APPROVED** [review]

File: data/news/articles/c0g992jmmkko.json (hunk 1 above)
Comment (alex-magana) [reply data/news/articles/c0g992jmmkko.json:898]: This was confirmed to have no effect when we migrated the Optimo
article page. I'l check with Andrew Anderson to confirm the same applies to CPS assets.

**Comment by pvaliani** [comment]
Moving this to RFT since it's been approved and all comments appear to be addressed. Nice work!

**Pull request merged**